proxy/src/main/java/net/md_5/bungee/connection/DownstreamBridge.java

```diff
@@ -400,14 +399,14 @@ public void handle(PluginMessage pluginMessage) throws Exception
                 if ( target.equals( "ALL" ) )
                 {
                     out.writeUTF( "ALL" );
-                    out.writeUTF( Util.csv( bungee.getPlayers() ) );
+                    out.writeUTF( bungee.getPlayers().stream().map( ProxiedPlayer::getName ).collect( Collectors.joining( ", " ) ) );
                 } else
                 {
                     ServerInfo server = bungee.getServerInfo( target );
                     if ( server != null )
                     {
                         out.writeUTF( server.getName() );
-                        out.writeUTF( Util.csv( server.getPlayers() ) );
+                        out.writeUTF( bungee.getPlayers().stream().map( ProxiedPlayer::getName ).collect( Collectors.joining(", ") ) );
```

Comment:
server.getPlayers

Reply:
why not just leave it as Util.csv?

Reply:
I think this is more desirable because before we were assuming the toString of ProxiedPlayer/UserConnection was the player name before but now it's explicit through the stream.